### DISCUSSION TITLE
Can the warning about "no found environment" be turned into an error?

### DISCUSSION THREAD

**Discussion by seifertm** [ymyzk/tox-gh-actions#154] [Q&A] [answered]
**Question**
When no environment is configured, tox-gh-actions gracefully degrades to a no-op and [prints a warning](https://github.com/ymyzk/tox-gh-actions/blob/v2.11.0/src/tox_gh_actions/plugin.py#L73-L77). Is there a way to turn this warning into an error?

**Context**
[pytest-asyncio uses tox-gh-actions.](https://github.com/pytest-dev/pytest-asyncio/blob/v0.20.3/.github/workflows/main.yml) We run the tests against different CPython versions and forgot to update the `3.11-dev` CI environment with `3.11`. Nobody noticed for quite a few weeks that the [tests for Python 3.11 were not actually run.](https://github.com/pytest-dev/pytest-asyncio/actions/runs/3648127098/jobs/6161183418)

I would prefer to get an error that results in a failing CI pipeline. Is there some configuration to make this possible?

**Comment by ymyzk** [comment, marked as answer]
Hello, thanks for filing a ticket. I think this ticket is mostly the same as #66 and I just added an option to make this warning an error. Please have a look at the other ticket and the updated README for more details. Thanks.

**Comment by seifertm** [comment]
Sorry for filing a duplicate.

Thank you for the pointers, this should do the trick.
